src/main/java/frc/trigon/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,131 @@
+package frc.trigon.robot.subsystems.arm;
+
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+public class Arm extends SubsystemBase {
+
+    private final CANSparkMax
+            MASTER_ELEVATOR_MOTOR = ArmConstants.MASTER_ELEVATOR_MOTOR,
+            FOLLOWER_ELEVATOR_MOTOR = ArmConstants.MASTER_ELEVATOR_MOTOR,
+            MASTER_ANGLE_MOTOR = ArmConstants.MASTER_ELEVATOR_MOTOR,
+            FOLLOWER_ANGLE_MOTOR = ArmConstants.FOLLOWER_ELEVATOR_MOTOR;
+    private final TalonSRX ELEVATOR_TALON_SRX_ENCODER = ArmConstants.ELEVATOR_TALON_SRX_ENCODER;
+    private final static Arm INSTANCE = new Arm();
+
+    private TrapezoidProfile
+            elevatorMotorProfile = null,
+            angleMotorProfile = null;
+    private double lastAngleMotorProfileGenerationTime,lastElevatorMotorProfileGenerationTime;
+
+    public static Arm getInstance() {
+        return INSTANCE;
+    }
+
+    private Arm(){}
+
+
+
+    void generateAngleMotorProfile(Rotation2d targetAngle){
```

Comment:
Extra newlines
Also, install the save actions - xdev edition plugin and configure it like I said in the group.